GCC 9.4 warning on arm:
```
[ 77%] Building CXX object modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/phasecorr_iterative.cpp.o
/home/alexander/opencv/modules/imgproc/src/moments.dispatch.cpp:13:19: warning: no previous declaration for ‘void (* cv::getMomentsInTileFunc(int))(const cv::Mat&, double*)’ [-Wmissing-declarations]
   13 | MomentsInTileFunc getMomentsInTileFunc(int depth)
      |                   ^~~~~~~~~~~~~~~~~~~~
[ 77%] Building CXX object modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/pyramids.cpp.o
```

--- a/modules/imgproc/src/moments.dispatch.cpp
+++ b/modules/imgproc/src/moments.dispatch.cpp
@@ -10,6 +10,8 @@
 
 namespace cv {
 
+MomentsInTileFunc getMomentsInTileFunc(int depth);
+
 MomentsInTileFunc getMomentsInTileFunc(int depth)
 {
     CV_CPU_DISPATCH(getMomentsInTileFunc, (depth),

